applications/luci-app-speedtest/root/usr/share/rpcd/acl.d/luci-app-speedtest.json

```diff
@@ -0,0 +1,10 @@
+{
+	"luci-app-speedtest": {
+		"description": "Grant access to speedtest",
+		"write": {
```

Comment:
Is it a problem to use `		"read": {` instead?

Reply:
I haven't tried it yet, but I'll give it a shot. Perhaps it'll work out.